### ISSUE TITLE
cifar queues problem

### ISSUE THREAD

**Issue by yulin01** [deepmind/learning-to-learn#32]
This project would not work with cifar

New problems can be implemented very easily. You can see in train.py that
the meta_minimize method from the MetaOptimizer class is given a function
that returns the TensorFlow operation that generates the loss function we want
to minimize (see problems.py for an example).

It's important that all operations with Python side effects (e.g. queue
creation) must be done outside of the function passed to meta_minimize. The
cifar10 function in problems.py is a good example of a loss function that
uses TensorFlow queues.

**Comment by LyyLqx** [comment]
This project would not work with cifar


New problems can be implemented very easily. You can see in `train.py` that
the `meta_minimize` method from the `MetaOptimizer` class is given a function
that returns the TensorFlow operation that generates the loss function we want
to minimize (see `problems.py` for an example).

It's important that all operations with Python side effects (e.g. queue
creation) must be done outside of the function passed to `meta_minimize`. The
`cifar10` function in `problems.py` is a good example of a loss function that
uses TensorFlow queues.
